service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -121,19 +129,33 @@ public void process(Exchange exchange) {
                     (HealthDataAssessment) exchange.getProperty("evidence");
                 if (evidence.getErrorMessage() != null) {
                   log.error("Health Assessment Failed");
+                } else {
+                  exchange.setProperty(
+                      "sufficientForFastTracking", evidence.isSufficientForFastTracking());
+                  log.info(
+                      " MAS Processing >> Sufficient Evidence >>> "
+                          + evidence.isSufficientForFastTracking());
+                  var masTransferObject =
+                      new MasTransferObject(claimPayload, assessment.getEvidence());
+                  exchange.getMessage().setBody(masTransferObject);
                 }
-                // FAKE IT: Let's pretend Health assessment passed
-                var masTransferObject =
-                    new MasTransferObject(claimPayload, assessment.getEvidence());
-                exchange.getMessage().setBody(masTransferObject);
               }
             })
-        //
-        // TODO: call pcOrderExam in the absence of evidence
-        // TODO: Call claim status update
         .process(FunctionProcessor.fromFunction(MasCollectionService::getGeneratePdfPayload))
-        .to(PrimaryRoutes.ENDPOINT_GENERATE_PDF);
+        .to(PrimaryRoutes.ENDPOINT_GENERATE_PDF)
+        // Call pcOrderExam in the absence of evidence
+        .process(
+            new Processor() {
+              @Override
+              public void process(Exchange exchange) throws Exception {
+                MasAutomatedClaimPayload claimPayload =
+                    (MasAutomatedClaimPayload) exchange.getProperty("claim");
+                exchange.getMessage().setBody(claimPayload);
+              }
+            })
+        .to(orderExamEndpoint); // Call Order Exam;
```

Comment:
Is this supposed to happen after generate PDF? According to this diagram, the order exam path also goes to PDF: https://github.com/department-of-veterans-affairs/abd-vro/wiki/VRO-v2-Roadmap

I may be reading wrong though as I am not very familiar with the actual requirements.

Reply:
Yes, generate PDF will just persist the PDF in the DB, we have to retrieve and upload to the e-folder as part of the final step